nimble-csp.yaml

```diff
@@ -0,0 +1,58 @@
+# Configuration to deploy the HPE Nimble Storage CSP service
```

Comment:
Can you add "for Velero" here to reflect this YAML represents the Velero Edition?

Reply:
will do